bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/internal/profiles/RawRockerToDimmerProfile.java

```diff
@@ -0,0 +1,129 @@
+/**
+ * Copyright (c) 2014,2017 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
```

Comment:
Just out of curiosity: Was this empty line automatically generated by the license plugin? If so, we seem to have a bug there...

Reply:
No, it wasn't, I copy/pasted the header from another file and might have added the extra line manually.
I prefer doing it manually, since the automated script also adds headers to files that are unrelated to what I am working on and then I have to manually exclude those things from commits, as has happened in a previous PR of mine.

Reply:
Surely it is ok if you do it manually. You just should not forget to do so - the files still contain the empty line...